pyart/default_config.py

```diff
@@ -1096,3 +1096,105 @@
     'chl': chl_field_mapping,
     'gamic': gamic_field_mapping,
 }
+
+
+def velocity_limit(container=None, selection=0):
+    import pyart
+    if isinstance(container, pyart.core.Radar):
+        try:
+            if selection >= 0 and selection < container.nsweeps:
+                vel = container.get_nyquist_vel(selection,
+                                                check_uniform=False)
+            else:
+                vel = container.get_nyquist_vel(0, check_uniform=False)
+            return (-vel, vel)
+        except LookupError:
+            return (-30., 30.)
+    else:
+        return (-30., 30.)
+
+
+def spectrum_width_limit(container=None, selection=0):
+    import pyart
+    if isinstance(container, pyart.core.Radar):
+        try:
+            if selection >= 0 and selection < container.nsweeps:
+                vel = container.get_nyquist_vel(selection,
+                                                check_uniform=False)
+            else:
+                vel = container.get_nyquist_vel(0, check_uniform=False)
+            return (0, vel)
+        except LookupError:
+            return (0, 30.)
+    else:
+        return (0, 30.)
+
+# map each field to a colormap
+
+DEFAULT_FIELD_COLORMAP = {
+    # field name : colormap
+    'reflectivity': 'pyart_NWSRef',
+    'corrected_reflectivity': 'pyart_NWSRef',
+    'total_power': 'pyart_StepSeq25',
```

Comment:
pyart_NWSRef would be alright here since total power is another reflectivity measurement.